tests/foreman/virtwho/test_api_virtwho.py

```diff
@@ -67,6 +71,26 @@ def _make_virtwho_configure(self):
             args[u'hypervisor_password'] = self.hypervisor_password
         return args
 
+    def _try_to_get_guest_bonus(self, hypervisor_name, sku):
```

Comment:
Could you try to use `wait_for_tasks` to check the vdc_id is ready or not?
I see there is function: `from robottelo.api.utils import wait_for_tasks`

Reply:
> Could you try to use `wait_for_tasks` to check the vdc_id is ready or not?
> I see there is function: `from robottelo.api.utils import wait_for_tasks`

I tried `wait_for_tasks`, such as:
`wait_for_tasks(search_query='label=Actions::Katello::Host::AttachSubscriptions ',
                       search_rate=1,
                       max_tries=10,
                       poll_timeout=10
                       )`
It would return all results about `AttachSubscriptions`, still need to filter it, so I would like to use `wait_for` as Mirek's suggustion and it works fine.